src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const result = days * 40;
+
+  if (days >= 3 && days < 7) {
+    return result - 20;
+  }
+
+  if (days >= 7) {
+    return result - 50;
+  }
```

Comment:
you can switch if statements places and don't write long condition
```suggestion
  if (days >= 7 ) {
    return result - 50;
  }

  if (days >= 3) {
    return result - 20;
  }
```

Reply:
agree